### DISCUSSION TITLE
xDrip+ Sync Follower to display raw data

### DISCUSSION THREAD

**Discussion by psonnera** [NightscoutFoundation/xDrip#2990] [Ideas]
When changing my data source from Sync Follower to Libre Bluetooth I can see raw data. Could this also be available for the follower? I mean, data is already there!
![Screenshot_20230809_114134_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/33809361/5c05874a-62c5-4375-b720-e349138aa589)
![Screenshot_20230809_114154_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/33809361/7c3d8534-d2b6-43ba-8472-33e548b7d5c7)

**Comment by Navid200** [comment]
I'm not sure what you mean by "data is already there".

Do you know that the raw values are being sent by xDrip sync through the Google servers to the follower?

**Reply by psonnera** [reply]
Yes indeed, this is what I meant by "already there". The values are sent by sync but not displayed in follower data source.

**Reply by Navid200** [reply]
Can you show me how I can verify, the same way you have verified, that the raw values are sent by sync?
I mean if you use xDrip as a collector, and you see the raw values, that doesn't prove that they are sent by sync.
It's quite possible I am overlooking something.  But, I need to understand this before I can triage it.

**Reply by psonnera** [reply]
@Navid200 this is a follower phone. It has no way to get raw if not by Sync. To reproduce: use a sensor showing raw data on a master phone, setup another phone as sync follower and enable raw data display. Wait for data to show up. Switch the follower to a bluetooth sensor data source.  Raw data displays. Note: concern regarding raw data that in my case is from a Libre sensor and might be handled differently from the G6 raw data internally.
